clients/rospy/src/rospy/timer.py

```diff
@@ -70,14 +85,7 @@ def sleep(self):
         @raise ROSTimeMovedBackwardsException: if ROS time is set
         backwards
         """
-        curr_time = rospy.rostime.get_rostime()
```

Comment:
With the removal of the local variable `curr_time` line 93 (https://github.com/ros/ros_comm/pull/588/files#diff-098242a094fe0fb6b870f1949712b77fR93) now fails.